.github/workflows/ci.yml

```diff
@@ -43,16 +43,17 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     strategy:
+      fail-fast: false
```

Comment:
is this really required? by reading the docs it seems it's mostly used to get quicker feedback when changing the workflow, so you can iterate faster on the CI changes, but when it's ready (meaning the CI config is ok) it should be "on", right?

Reply:
When this is ON if we have a failure only on macos + go version X it will stop/abort all other parallel builds on the build matrix. So we get a faster failure but we lose the information if a specific change is going to work on other combinations of OS + Go version. The main drawback is only taking longer, IMO it is more desireable to in the face of a failure be sure if the failure is on all supported OS/versions than being faster.

Reply:
+ as explored with @i4ki GH doesn't provide a proper way to run just an aborted step, so there is no good way to check if an aborted step will succeed or fail (it always runs all steps or all failed steps, but that keeps the aborted step being aborted).